doc/src/manual/arrays.md

```diff
@@ -528,10 +528,8 @@ applied to arrays (and combinations of arrays and scalars) as a
 operations like `<`, *only* the `.<` version is applicable to arrays.)
 
 1. Unary arithmetic -- `-`, `+`, `!`
-2. Binary arithmetic -- `+`, `-`, `*`, `/`, `\`, `^`, `.^`, `div`, `mod`
+2. Binary arithmetic -- `+`, `-`, `*`, `/`, `\`, `^`, `.^`
```

Comment:
Good question. With more thought, the beginning of this section is somewhat confusing: `*`, `/`, `\`, and `^` only operate elementwise over arrays when one of the arguments is a scalar (as noted below); "vectorized" is not an appropriate description of their behavior outside of those cases. And in a meaningful sense `+` and `-` are "whole-array operations" that happen to be elementwise. So perhaps this section should be retitled "Array and Vectorized Operators and Functions", enumerate the "whole-array operations" first, then note that some of those "whole-array operations" operate elementwise when one argument is a scalar, and then note that all scalar operators have a dot form et cetera. Thoughts on that reorg / clarification? Thanks!